doc/api/esm.md

```diff
@@ -801,6 +801,18 @@ property:
 
 ## Differences Between ES Modules and CommonJS
 
+### Absolute file paths don't work on Windows
+
+Note that `import` accepts _absolute specifiers_. These are similar to _absolute paths_ on linux and MacOS, and therefore they are also accepted. But on Windows, importing an _absolute path_ to a file will not work. Therefore, to ensure cross platform compatibility, be sure to turn your file paths into urls, like so:
+
+```
+import { pathToFileURL } from 'url';
+
+const fileUrl = pathToFileURL(...your path here...).href;
```

Comment:
```suggestion
const fileUrl = pathToFileURL('c:\\some\\path\\here').href;
```

Better to have an actual path. This won't fail linting either.

Reply:
**ESM** requires the file extension:
```suggestion
const fileUrl = pathToFileURL('c:\\some\\path\\here.js').href;
```

Reply:
```suggestion
const fileUrl = pathToFileURL('c:\\path\\to\\file.js').href;
```